share/completions/black.fish

```diff
@@ -0,0 +1,23 @@
+complete -c black -s c -l code -d 'Code to format'
+complete -c black -s l -l line-length -d 'Characters per line'
+complete -c black -s t -l target-version -d 'Python versions supported by output'
```

Comment:
This also needs `-x`. We can maybe even compute the possible choices based on the output of `black --help`, but that's not necessary.